MoreHtml2/README.md

```diff
@@ -4,6 +4,8 @@ This continuation of the More HTML Lesson allows the students to practice the co
 ## Icebreaker
 For this session, play a game of [Drawphone](../Icebreakers/DrawphoneIcebreaker.md).
 
+Alternatively, break the students into groups and have them compete in [web design project](./Web%20Design%20Requirements/).
```

Comment:
small thing but I'll remove the spaces from the folder name just so we don't have to use `%20` in the link!